### DISCUSSION TITLE
how to configure mfsbdev?

### DISCUSSION THREAD

**Discussion by eleaner** [moosefs/moosefs#467] [Q&A]
Is there a place where I can find more? Maybe an example?

I run 
`sudo mfsbdev start -H mfsmaster -P 9421 -S nbd`

that assumes nbd is a folder in the root of my mfs
that seems to work
`mfsmaster accepted connection with parameters: read-write,restricted_ip,admin ; root mapped to root:root ; setgoal limited to (2:9)`

however, when I try to do mapping
`sudo mfsbdev map -f bismuth -s 250G`
I get
`can't find free NBD device`
 
that assumes that the file with the device will be called "bismuth" and I want it to be 250G sie

**Comment by eleaner** [comment]
possibly the only thing I missed was
`sudo modprobe nbd`

**Reply by chogata** [reply]
Did you manage to start your device when you loaded the nbd module? mfsbdev is very simple, all the available options are described in man page (it mentions the need for nbd module to be loaded).

**Comment by eleaner** [comment]
yup, all worked beautifully
